Flagsmith.Client.Test/FlagsmithTest.cs

```diff
@@ -112,11 +112,15 @@ public async Task TestGetIdentityFlagsUsesLocalEnvironmentWhenAvailable()
                 StatusCode = System.Net.HttpStatusCode.OK,
                 Content = new StringContent(Fixtures.JsonObject.ToString())
             });
+
+            var traits = new List<ITrait> { new Trait("foo", "bar") };
```

Comment:
Added traits here to ensure that there is logic to cover instantiating the identity model with traits.